src/Idris/Core/Elaborate.hs

```diff
@@ -139,13 +139,14 @@ runElab a e ps = runStateT e (ES (ps, a) "" Nothing)
 execElab :: aux -> Elab' aux a -> ProofState -> TC (ElabState aux)
 execElab a e ps = execStateT e (ES (ps, a) "" Nothing)
 
-initElaborator :: Name -> Context -> Ctxt TypeInfo -> Type -> ProofState
+initElaborator :: Name -> Context -> Ctxt TypeInfo -> Int -> Type -> ProofState
```

Comment:
Can we clean up/document these args somehow? Especially with the `Int` now it's not clear from the signature how I should call this.